mapie/multi_label_classification.py

```diff
@@ -242,6 +272,33 @@ def _check_delta(self, delta: Optional[float]):
                 + "even if the delta is not ``None``, it won't be"
                 + "taken into account"
             )
+        if (self.method == "ltt"):
+            if delta is None:
+                raise ValueError(
+                    "Invalid delta. "
+                    "delta cannot be ''None'' when using "
+                    "LTT method."
+                )
+            elif ((delta <= 0) or (delta >= 1)):
+                raise ValueError(
+                    "Invalid delta. "
+                    "delta must be in ]0, 1["
+                )
+
+    def _check_valid_index(self, alpha: NDArray):
+        """
+        Check if the valid index is empty.
+        If it is, we should warn the user that for the alpha value
+        and delta level chosen, LTT will return no results.
+        The user must be less inclined to take risks or
+        must choose a higher alpha value.
+        """
+        for i in range(len(self.valid_index)):
+            if self.valid_index[i] == []:
+                warnings.warn(
+                    "Warning: LTT method has returned an empty sequence"
+                    + " for alpha=" + str(alpha[i])
+                )
```

Comment:
QUESTION: Is it possible to have too many warning?
```python
if self.method == 'ltt' and not all(self.valid_index):
```
Why doesn't this suggestion satisfy you?

Reply:
I want the user to know which value of alpha have given him an empty sequence. With the use of all function, he will only have a warning without knowing which alpha give this warning.